galata/test/jupyterlab/texteditor.test.ts

```diff
@@ -20,30 +20,20 @@ test.describe('Text Editor Tests', () => {
     const imageName = 'text-editor-rulers.png';
     await page.menu.clickMenuItem('File>New>Text File');
 
-    await page.waitForSelector(`[role="main"] >> text=${DEFAULT_NAME}`);
+    await page.getByRole('main').getByText(DEFAULT_NAME).waitFor();
 
     await page.menu.clickMenuItem('Settings>Settings Editor');
 
-    await page.waitForSelector('text=Text Editor');
-    await page.click('text=Text Editor');
+    await page
+      .getByRole('tab', { name: 'Editor Text Editor' })
```

Comment:
```suggestion
      .getByRole('tab', { name: 'Editor Text Editor' })
```

Is this value correct?